test/endpoints/tchannel/baz/baz_simpleservice_method_call_test.go

```diff
@@ -42,6 +42,9 @@ func TestCallTChannelSuccessfulRequestOKResponse(t *testing.T) {
 		KnownTChannelBackends: []string{"baz"},
 		TestBinary:            util.DefaultMainFile("example-gateway"),
 		ConfigFiles:           util.DefaultConfigFiles("example-gateway"),
+		TChannelClientMethods: map[string]string{
```

Comment:
It was needed to avoid nil pointer exceptions for client method loggers in tests. I can default to a client logger if the method logger is not present.

Reply:
I can't avoid passing the tchannel client methods as part of the test gateway options, since the tchannel client is created/called on the test gateway (not actual gateway), so the test gateway needs know the client methods when setting up the method logger and metrics for the tchannel client.